package.json

```diff
@@ -500,17 +500,24 @@
             "markdownDescription": "Enable Ansible Lightspeed with watsonx Code Assistant inline suggestions.",
             "order": 2
           },
+          "ansible.lightspeed.suggestions.waitWindow": {
+            "scope": "resource",
+            "type": "number",
+            "default": 0,
```

Comment:
Do we want to set the default to a small value to start with?

Reply:
> Do we want to set the default to a small value to start with?

I brought this on our team's Slack channel and Marty and Robin agreed to use 0 as the default value. I think it is safe not to change the current default behavior and making the response time longer by default does not seem to be a good idea even if it is a few hundred milli-seconds.